plutus-core/untyped-plutus-core/Language/UntypedPlutusCore/Subst.hs

```diff
@@ -0,0 +1,43 @@
+module Language.UntypedPlutusCore.Subst
+    ( termSubstFreeNamesA
```

Comment:
Do we need to export `termSubstFreeNamesA` (also in the typed version)? It doesn't appear to be used elsewhere.

Reply:
The `*A` functions exist, because some of them are used somewhere (in tests IIRC). For uniformity we export all of them, even if some are unused. And it generally makes sense to export stuff that is not used _at the moment_, but can be used later. Basically API and how the API is used are two very different things.